sfa_dash/templates/forms/fx_data_download_form.html

```diff
@@ -0,0 +1,70 @@
+{% extends "dash/data.html" %}
+{# {% set sidebar = True %} #}
+{% set page_title = 'Download Forecast data' %}
+{% block content %}
+{{ metadata | safe }}
+<h3>Download data for Forecast 'DA Power'</h3>
+{#<p><b>Available Date Range:</b> <i>2016-01-01T12:00Z</i> to <i>2019-01-01T12:00Z</i></p>#}
+<form action="{{ url_for('forms.download_forecast_data', uuid=uuid) }}" method="post" id="forecast-download">
+  <div class="form-group">
+	<div class="form-element">
+	  <label for="start">Start time:</label>
+	  <input type="datetime-local" id="start" name="period-start">
+    </div>
+    <div class="form-element">
+	  <label for="end">End time:</label>
+	  <input type="datetime-local" id="end" name="period-end">
+    </div>
+	<div class="form-element">
+	  <label>Timezone</label>
+	  <select required name="timezone" class="form-control">
+        <option value="utc">UTC</option>
+      </select>
+	</div>
+	<div class="form-element full-width">
+	  <label for="format">Format</label><br/>
+	  <input type="radio" data-toggle="collapse" data-target=".download-example" name="format" value="text/csv" checked>CSV</input>
+	  <input type="radio" data-toggle="collapse" data-target=".download-example" name="format" value="application/json">JSON</input>
+	</div>
+	<div class="form-element full-width">
+	  <div class="download-example csv show">
+	  <p>CSV downloads will have the following format.</p>
+	  <pre class="example-text">
+# name: DA Power
+# forecast_id: f79e4f84-e2c3-11e8-9f32-f2801f1b9fd1
```

Comment:
route is `forecasts` with an s. should it be consistent?

Reply:
or just `id` for consistency with the json format

Reply:
I think forecast_id, observation_id and site_id make more sense as the id labels for single objects. The json should show forecast_id as well.
I'll have to fix the discrepancies between the API and data model again, as the json endpoints don't include name or links, and the csv endpoint does not write metadata as a header comment.